viewer/packages/modeldata-api/src/applyDefaultModelTransformation.ts

```diff
@@ -15,6 +15,10 @@ export function applyDefaultModelTransformation(matrix: THREE.Matrix4, format: F
       matrix.premultiply(cadFromCdfToThreeMatrix);
       break;
 
+    case File3dFormat.GltfCadModel:
+      matrix.premultiply(cadFromCdfToThreeMatrix);
+      break;
```

Comment:
Move `case` clause up to the one above, so the code becomes
```suggestion
    case File3dFormat.RevealCadModel:
    case File3dFormat.GltfCadModel:
      matrix.premultiply(cadFromCdfToThreeMatrix);
      break;
```